src/ui.button-toggle/UToggle.vue

```diff
@@ -86,6 +86,11 @@ const { toggleLabelAttrs, itemsAttrs, itemAttrs } = useUI<Config>(defaultConfig)
     v-bind="toggleLabelAttrs"
     :data-test="dataTest"
   >
+    <template #label>
+      <!-- @slot Use it to add something instead of label. -->
```

Comment:
Use this to add custom content instead of the label. - change for all translations.
and label slot props should be documented.